Tests/DependencyInjection/PrestaSitemapExtensionTest.php

```diff
@@ -17,4 +17,31 @@ public function testDumperAliasIsSet()
 
         self::assertTrue($containerBuilder->hasAlias('Presta\SitemapBundle\Service\DumperInterface'));
     }
+
+    public function testAlternate()
```

Comment:
This is indeed the good direction, but we must add more tests for that section : maybe a data provider with several use cases : 
- no config for the bundle
- disabled alternate section
- enabled alternate section with config